### ISSUE TITLE
Garbled Audio

### ISSUE THREAD

**Issue by stvaruzek** [ShareX/ShareX#4232]
Hi Folks,

first and foremost, thank you for ShareX as I was evaluating bunch of screen capturing tools and ShareX simply rocks!!!

But I have one issue which drives me crazy... I made today a few video tutorials (cca 5 minutes long) but almost always recorded audio gets garbled in the middle / end of the video.

What I mean is that when I do replay the captured video there are sections where you cannot understand voice. I can hear beginning of a word and then it ends with other word.

Looks like some audio frames were skipped during recording... No idea...

You can imagine how I feel when I did five times the same tutorial and then I discovered that the audio is garbled again 😄 

Any idea? I am recording on my 4k monitor and here are my ffmpeg settings:
`-y -rtbufsize 150M -f gdigrab -framerate 30 -offset_x 0 -offset_y 0 -video_size 3840x2160 -draw_mouse 1 -i desktop -f dshow -i audio="Headset Microphone (Plantronics C320-M)" -c:v libx264 -r 30 -preset ultrafast -tune zerolatency -crf 28 -pix_fmt yuv420p -movflags +faststart -c:a aac -strict -2 -ac 2 -b:a 128k "output.mp4"`

The only workaround I can think of is to disable audio recording in ShareX and record voice it in other tool and then merge with ffmpeg  but that's really the last resort.

That's why I am askling here for help / suggestions how to fix the issue.

Thank you.

**Comment by stvaruzek** [comment]
I have sorted it out. The root cause was obviously my 4K monitor. When I played with ffmpeg directly on command line and I set the -rtbufsize to max 2048M I saw that frames were skipped after a while.

Then I tried to change resolution on my monitor from 3840x21060 to 1920x1080 and the issue got solved. However the video quality was poor. Simply sharpness went down when I switched to lower resolution and it was visible in the videos. 

So at the end I managed to do it on my laptop which is 1920x1080 and videos look good also on the 4K monitor.
